packages/util-logger/src/util-logger.ts

```diff
@@ -47,6 +45,15 @@ const buildLoggerMessageArgs = ({title, message}) => {
   ]
 }
 
+const getLoggerLevel = () => {
+  try {
+    const loggerLevel = parseInt(process?.env?.LOGGER_LEVEL)
```

Comment:
Using process.env for the logger level is a breaking change and additionally process.env doesn't exist in browser environments. Is there a reason we need to change this?

Reply:
I'm assuming it's because of the circular dependency... Is there any way we can get around this with some sort of dependency injection instead?

Reply:
this was due to a circular dependency between `util-logger` and `config`

Reply:
restored